ckanext/opendata_theme/opengov_custom_theme/templates/user/new.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+{% block help_inner %}
+  <h2 class="module-heading">{{ _('Why Sign Up?') }}</h2>
+  <div class="module-content">
+    <p>{% trans %}Follow datasets to receive notifications when new activity occurs{% endtrans %}</p>
```

Comment:
I think we never talked about this, but does Open Data support any kind of i18n?